AppDashboard/lib/app_dashboard_data.py

```diff
@@ -171,6 +173,77 @@ def __init__(self, helper=None):
+                            "is_admin_panel":True,
+                            "template":"apps/memcache.html"},
+          "upload_app":{"title":"Upload Application",
+                        "link":"/apps/new",
+                        "template":"apps/new.html"},
+          "delete_app":{"title":"Delete Application",
+                        "link":"/apps/delete",
+                        "template":"apps/delete.html"},
+          "relocate_app":{"title":"Relocate Application",
+                          "link":"/apps/relocate",
+                          "template":"apps/relocate.html"},
+          "manage_users":{"title":"Manage Users",
+                          "link":"/authorize",
+                          "is_admin_panel":True,
+                          "template":"authorize/cloud.html"},
+          "logging":{"title":"Log Viewer",
+                     "link":"/logs",
+                     "template":"logs/main.html"},
+          "monit":{"title":"Monit",
+                   "link":self.get_monit_url()},
+          "taskqueue":{"title":"TaskQueue",
+                       "link":self.get_flower_url()},
+          "app_console":{"title":"Application Statistics",
+                         "template":"apps/console.html",
+                       "link":"/apps/"}
+        }
+        if(user_info.can_upload_apps):
+          LOOKUP_DICT["app_management"] = {"App Management":
+                                          [{"upload_app": LOOKUP_DICT[
+                                            "upload_app"]},
+                                           {"delete_app": LOOKUP_DICT[
+                                             "delete_app"]},
+                                           {"relocate_app": LOOKUP_DICT[
+                                             "relocate_app"]}]}
+        if(user_info.is_user_cloud_admin):
+          LOOKUP_DICT["appscale_management"] = {"AppScale Management":
+                                               [{"cloud_stats": LOOKUP_DICT[
+                                                 "cloud_stats"]},
+                                                {"manage_users": LOOKUP_DICT[
+                                                  "manage_users"]}]}
+        if(user_info.owned_apps or user_info.is_user_cloud_admin):
+          LOOKUP_DICT["debugging_monitoring"] = {"Debugging/Monitoring":
+                                                [{"monit": LOOKUP_DICT["monit"]},
+                                                 {"taskqueue": LOOKUP_DICT[
+                                                   "taskqueue"]},
+                                                 {"logging": LOOKUP_DICT[
+                                                   "logging"]},
+                                                 {"app_console": LOOKUP_DICT[
+                                                   "app_console"]}]}
+        return LOOKUP_DICT
+    except Exception as err:
```

Comment:
Its catching the user_info exception. Or at least I assume so, every other function that made the first if statements and tried to retrieve user_info catch an exception. This method needs to check user_info variables otherwise it will not have any of the navigation keys. That was the bug I chased down last week.